### PR TITLE
Avatar and alert component created.

### PR DESCRIPTION
**Pull request by ArpitPatni**
Hi 
I have created avatar and alert component.
Could you please review my code.

### CODE REVIEW

**Comment by anurag-kkumarr** [comment]
Looks good to merge.

File: Styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.alert-primary {
+    background-color: #7FD2E6;
+    color: #29438b;
+}
```
Comment (vikrantkode) [inline Styles/alert.css]: use variable while giving color or background color

File: Styles/avatar.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.main-title{
+    margin-left: 200px;
+    padding: 40px;
+}
+
+
+
+.avatar-heading{
```
Comment (vikrantkode) [inline Styles/avatar.css]: Use proper spaces between the classes

File: Styles/middle.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    color:var(--white-color);
+    height:500px;
+    padding-left: 120px;
+}
+.main-section-container{
+    display:flex;
+    flex-direction: row wrap;
+    justify-content: space-between;
+    align-items: center;
+}
```
Comment (vikrantkode) [inline Styles/middle.css]: remove commented code if not needed

**Review by vikrantkode — CHANGES_REQUESTED** [review]
in Landing Page Ui
- Fix  footer to bottom

File: Styles/middle.css (hunk 3 above)
Comment (ArpitPatni) [reply Styles/middle.css]: Thank you for suggestion.I have made the changes.

File: Styles/avatar.css (hunk 2 above)
Comment (ArpitPatni) [reply Styles/avatar.css]: Thank you for suggestion.I have made the changes.

File: Styles/alert.css (hunk 1 above)
Comment (ArpitPatni) [reply Styles/alert.css]: I wouldn't be using alert color in other parts of the project and there are so many colors inside alerts component , that's why I have not used variable for them , other than that variables are being used properly.

**Pull request merged**